contrib/mac/framework/julia-config.jl

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env julia
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+const options =
+[
+    "--cflags",
+    "--ldflags",
+    "--ldlibs"
+];
```

Comment:
add the symlinks until it actually becomes a problem, but build the tooling in a generic way that doesn't have to rely on them? if we introduce a variable for the name of libjulia (which seems much less framework specific to me, so useful to add) then I still suspect many people would miss using it.